lib/Common/Memory/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ add_library (Chakra.Common.Memory
     ArenaAllocator.cpp
 
     # xplat-todo: This is needed for allocating jitted code memory
```

Comment:
Remove this comment now since this is no longer something that needs to be done :smile: